script/test

```diff
@@ -46,3 +46,7 @@ echo "==> test: setting up node_modules…"
 echo
 echo "==> test: executing \`cypress run\`…"
 (cd "${temp_path}" && npx cypress run)
+
+
+
+ 
```

Comment:
Always review all your changes as you stage them for commit, and again under the Files Changed view once you open your PR, and try to minimize extraneous changes